src/coreclr/tools/Common/CommandLineHelpers.cs

```diff
@@ -427,5 +426,26 @@ public static bool TryReadResponseFile(string filePath, out IReadOnlyList<string
             newTokens = null;
             return false;
         }
+
+        private sealed class CustomizedHelpAction : SynchronousCommandLineAction
+        {
+            private readonly HelpAction _helpAction;
+            private readonly Action<ParseResult> _customizer;
+
+            public CustomizedHelpAction(HelpOption helpOption, Action<ParseResult> customizer)
+            {
+                _helpAction = (HelpAction)helpOption.Action;
+                _customizer = customizer;
+            }
+
+            public override int Invoke(ParseResult parseResult)
+            {
+                int result = _helpAction.Invoke(parseResult);
+
+                _customizer.Invoke(parseResult);
```

Comment:
```suggestion
                int result = _helpAction(parseResult);

                _customizer(parseResult);
```
Nit: Invoke is not necessary on regular delegate calls. I think the usual style is to omit it.